Automattic/woocommerce-payments
# Adjust eslint config to align with WC Core.

Fixes **RFC:** paJDYF-9nK-p2

#### Changes proposed in this Pull Request



* Adjust the config to remove the custom rule on yoda conditionals. This makes the config fallback to WCCore, which disallows yoda conditonals.
* Running the linter to make all conditionals fit the Core standard. This PR should contain **no** functional changes, just code formatting.







#### Testing instructions





* Check the code and confirm that the PR contains no changes that will functionally change how something is evaluated - the only changes will be reversing yoda conditionals.
* `===` comparisons will be the same, with the order of arguments reversed. `<` comparisons will be reversed. For example, `0 < length` will become `length > 0`.

-------------------


**Post merge**
